spec/support/factoy_bot.rb

```diff
@@ -0,0 +1,3 @@
+FactoryBot.define do
+  initialize_with { new attributes }
```

Comment:
I guess it is still used in some factories.

Reply:
Anything which has attributes setting via the constructor but not via a method - `Search` being the key culprit.